src/db-fixture-pg.ts

```diff
@@ -1,27 +1,17 @@
-import * as pg_db from "./db-pg";
-import * as shortid from "shortid";
+import * as pg_db from './db-pg';
+import * as shortid from 'shortid';
 
-
-export class PGFixture
-{
+export class PGFixture {
     tool1ID: number = null;
     tool1Name: string = null;
     tool2ID: number = null;
     tool2Name: string = null;
 
+    constructor() {}
 
-    constructor()
-    {
-    }
-
-
-    public addToDB(
-        db_frontend: pg_db.PG
-        ,success_callback: () => void
-    ): void
-    {
+    public addToDB(db_frontend: pg_db.PG, success_callback: () => void): void {
```

Comment:
Breaking out  parameters line by line, as in the original, is preferred. Commas can go to the end of the line.

Reply:
Prettier collapses to a single-line when it can.  That's the basic formatting algorithm: collapse-all and then progressively  expand to fit under the line-length.  There's not much control if you prefer always expanded to multi-line for function arguments such as this case. 😐